app/experimenter/nimbus-ui/README.md

```diff
@@ -185,6 +185,16 @@ if (message !== "success") {
 console.log("Retrieved record", record);
 ```
 
+## Access Visualization Data Locally
+
+The Results page renders experiment analysis data and can only be accessed when an experiment is complete, the analysis feature flag is on, and the analysis data returned from the visualization API endpoint is available.
+
+To see the Results page and data locally:
+
+1. Ensure your [Google Credentials](https://github.com/mozilla/experimenter#google-credentials) are configured
+2. Set `FEATURE_ANALYSIS=true` in your local `.env` file
+3. Create a Nimbus Experiment at `localhost/admin` with status "complete," a branch named "control", and another branch named "treatment"
+
```

Comment:
Do we want to include one of the dummy slugs as well?

Reply:
Ah oops you're right. I didn't realize it has to be that specific slug but it does.